### PR TITLE
Minimal coverage percentage & test output to console

### PR DESCRIPTION
**Pull request by rafaldziuryk**
Add minimal coverage percentage value to pass e.g. for gitlab
Add test output to console to show results

### CODE REVIEW

File: bin/test_coverage_advanced.dart [hunk 1]
```diff
@@ -48,12 +56,18 @@ Future main(List<String> arguments) async {
...
     print('Coverage report saved to "coverage/lcov.info".');
   });
   final lcov = File(path.join(packageRoot.path, 'coverage', 'lcov.info'));
   final lineCoverage = calculateLineCoverage(lcov);
   generateBadge(packageRoot, lineCoverage);
   final coveragePct = (lineCoverage * 100).floor();
   print('Overall line coverage rate: $coveragePct%.');
+  final minCoverage = int.parse(options["min-coverage"]);
+  if (coveragePct < minCoverage) {
+    print('Overall coverage $coveragePct is small then minimal coverage $minCoverage');
```
Comment (bobjackman) [inline bin/test_coverage_advanced.dart]: This should read `Overall coverage $coveragePct is less than minimum required coverage $minCoverage`

File: pubspec.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-name: test_coverage
+name: test_coverage_advanced
```
Comment (bobjackman) [inline pubspec.yaml]: I must have missed it somewhere... why are we changing the package name here?

Comment (rafaldziuryk) [reply pubspec.yaml]: i had problems with cache and change name - fixed for now ;) sorry

File: bin/test_coverage_advanced.dart (hunk 1 above)
Comment (rafaldziuryk) [reply bin/test_coverage_advanced.dart]: fixed

**Review by pulyaevskiy — CHANGES_REQUESTED** [review]
Thanks for taking time to submit this PR.
LGTM overall, though there is now a merge conflict. If you could resolve it I'll merge.

**Review by pulyaevskiy — APPROVED** [review]
LGTM, thanks again!

**Pull request merged**